homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -213,8 +212,7 @@ def __init__(self, config):
         """Initialize the instance of the view."""
         self.config = config
 
-    @core.callback
-    def get(self, request, username):
+    async def get(self, request, username):
```

Comment:
This won't make much difference, `create_list_of_entities` is still called within a single task ?

Reply:
Adjusted some more.  Only items left to fixup are the misbehaving integrations that are sucking up event loop time generating tracebacks and log spam

<img width="1011" alt="Screen Shot 2020-03-12 at 6 36 12 PM" src="https://user-images.githubusercontent.com/663432/76576589-848b2080-6490-11ea-8083-02c58ecc8423.png">

Reply:
Fixed